src/ChannelServer/World/Character.cs

```diff
@@ -622,12 +1094,28 @@ private void AddReferenceProperties()
 			this.Properties.Add(new RefFloatProperty(PropertyId.PC.UsedStat, () => this.UsedStat));
 			this.Properties.Add(new RefStringProperty(PropertyId.PC.AbilityPoint, () => this.AbilityPoints.ToString()));
 
-			this.Properties.Add(new RefFloatProperty(PropertyId.PC.SR, () => this.SplashRate));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.SR, () => this.AoeRatio));
 
 			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MINPATK, () => this.MinPAtk));
 			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MAXPATK, () => this.MaxPAtk));
 			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MINMATK, () => this.MinMAtk));
 			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MAXMATK, () => this.MaxMAtk));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MINPATK_SUB, () => this.MinPAtk_Sub));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MAXPATK_SUB, () => this.MaxPAtk_Sub));
+
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.CRTATK, () => this.CritATK));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.CRTHR, () => this.CritRate));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.HR, () => this.Accuracy));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.DR, () => this.Evasion));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.BLK, () => this.Block));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.BLK_BREAK, () => this.BlockPen));
+
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.DEF, () => this.Def));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MDEF, () => this.Mdef));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.CRTDR, () => this.CritRes));
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.SDR, () => this.AoeRatioDef));
+
+			this.Properties.Add(new RefFloatProperty(PropertyId.PC.MSPD, () => this.GetSpeed()));
```

Comment:
Are you sure MSPD is the speed? "SPD" surely stands for "speed", but what's the M?

Reply:
Movement speed, after that its appear in the F1 window.